### DISCUSSION TITLE
Return a user added item

### DISCUSSION THREAD

**Discussion by tomisanhues2** [stefvanschie/IF#284] [Q&A]
I want to,be able to allow users to input an item from their inventory into a designated slot in the GUI, after that I want to be able to retrieve the itemstack on a click event of a different slot so I can know what the item the user added is.

Example with slots:

Slot 1: User is allowed to add items, drops 1 dirt block

Slot 2: when this button is clicked, if slot 1 is not null. there's a way to get which item was entered in slot 1. A name or any information is enough

**Comment by stefvanschie** [comment]
You'd have to track which item the user puts in that slot yourself. Then, once another item is clicked, you can look at this information to see which item the user placed.
